tools/Config.in

```diff
@@ -0,0 +1,87 @@
+config TOOLS_BUILD_ALL
+	bool "Compile all host tools" if DEVEL
+	select TOOLS_BUILD_7z
+	select TOOLS_BUILD_b43-tools
+	select TOOLS_BUILD_cbootimage
+	select TOOLS_BUILD_dwarves
+	select TOOLS_BUILD_elftosb-sdimage
+	select TOOLS_BUILD_genext2fs
+	select TOOLS_BUILD_gmp
+	select TOOLS_BUILD_isl
+	select TOOLS_BUILD_kernel2minor
+	select TOOLS_BUILD_llvm-bpf
+	select TOOLS_BUILD_lz4
+	select TOOLS_BUILD_lzop
+	select TOOLS_BUILD_mpc
+	select TOOLS_BUILD_mpfr
+	select TOOLS_BUILD_sparse
+	select TOOLS_BUILD_squashfs-lzma
+	help
+	  Compile all host host tools even if not needed. This is needed to prepare a
+	  universal precompiled host tools archive to use in another buildroot.
+
+config TOOLS_BUILD_7z
+	bool
+	default y if TARGET_realtek
+
+config TOOLS_BUILD_b43-tools
+	bool
+	default y if SDK || PACKAGE_kmod-b43 || BRCMSMAC_USE_FW_FROM_WL
+	help
+	  Build tools for kmod-b43 and broadcom firmware
+
+config TOOLS_BUILD_cbootimage
+	bool
+	default y if TARGET_tegra
+
+config TOOLS_BUILD_dwarves
+	bool
+	default y if DWARVES
+
+config TOOLS_BUILD_elftosb-sdimage
+	bool
+	default y if TARGET_mxs
+
+config TOOLS_BUILD_genext2fs
+	bool
+	default y if TARGET_apm821xx || TARGET_gemini
+
+config TOOLS_BUILD_gmp
+	bool
+	default y if !EXTERNAL_TOOLCHAIN
+
+config TOOLS_BUILD_isl
+	bool
+	default y if !EXTERNAL_TOOLCHAIN && GCC_USE_GRAPHITE
+
+config TOOLS_BUILD_kernel2minor
+	bool
+	default y if USES_MINOR
+
+config TOOLS_BUILD_llvm-bpf
+	bool
+	default y if USE_LLVM_BUILD
+
+config TOOLS_BUILD_lz4
+	bool
+	default y if SDK || TARGET_INITRAMFS_COMPRESSION_LZ4
+
+config TOOLS_BUILD_lzop
+	bool
+	default y if SDK || TARGET_INITRAMFS_COMPRESSION_LZO
+
+config TOOLS_BUILD_mpc
+	bool
+	default y if !EXTERNAL_TOOLCHAIN
+
+config TOOLS_BUILD_mpfr
+	bool
+	default y if !EXTERNAL_TOOLCHAIN
+
+config TOOLS_BUILD_sparse
+	bool
+	default y if USE_SPARSE
+
+config TOOLS_BUILD_squashfs-lzma
```

Comment:
Please rename this into `TOOLS_BUILD_squashfs3-lzma`

Reply:
The configuration option does also not match the one you are using.
`tools/Makefile` uses `CONFIG_TOOLS_BUILD_squashfs-lzma-old`

Reply:
Did the package rename get merged? Otherwise it's not renamed yet.

Reply:
No not yet, but when you anyway create a new configuration option it would be nice to name it squashfs3

Reply:
These are not user-available anyway they could be named some unintelligible hash.  The intent was to name them what they enable (when it enables multiple packages).  Therefore `squashfs-lzma-old` because it enables `lzma-old` and `squashfs`.